### PR TITLE
Fixing duplicate user model - after PR#34

### LINKED ISSUE #45: Multiple user tables in DB
When we created the user model we created two tables instead of building on top of the django user

### PR DESCRIPTION
**Pull request by AshkenYariv**
close #45 
In database there were two tables for user, this PR fixes this bug
puts all the relevant attributes in a single django user with role as
additional info.

### CODE REVIEW

**Review by assafg95 — APPROVED** [review]

**Review by edenmaor — APPROVED** [review]

**Review by edenmaor — APPROVED** [review]

File: users/tests.py [hunk 1]
```diff
@@ -1,61 +1,100 @@
 import pytest
-from users.models import Role, User
-from django.contrib.auth.models import User as DjangoUser
+from users.models import User
 
 
 @pytest.fixture
-def create_user_1():
-    return User.create_user('user1', 'test@test.com', '132456', 'b', Role.STUDENT)
+def create_user_1() -> User:
```
Comment (shayakrach) [inline users/tests.py]: Change to user_1 only.

**Review by shayakrach — APPROVED** [review]
Looks good!

**Comment by yuvalk** [comment]
> Looks good!

looks good or still need to resolve the comment you raised with `def create_user_1`?

**Comment by AshkenYariv** [comment]
> 

@yuvalk yes fixing it now, will be up in 5 minutes.

**Comment by AshkenYariv** [comment]
> > Looks good!
> 
> looks good or still need to resolve the comment you raised with `def create_user_1`?

fixed. ready for merge.

**Pull request merged**